includes/hub/stores/class-subscriptions.php

```diff
@@ -58,18 +58,22 @@ public static function persist( Subscription_Changed $subscription ) {
 
 		Debugger::log( 'Persisting subscription ' . $subscription_id );
 
-		$local_id          = self::get_local_id( $subscription );
-		$subscription_data = self::fetch_data_from_api( $subscription );
```

Comment:
`$subscription_data` is removed here, but still used on line 86.

Reply:
It seems I forgot to update that to add the products. Fixed in https://github.com/Automattic/newspack-network/pull/123/commits/dbc12f1f4a49955abcf544e5778eb9b3e68fa174

Also noticed that I forgot to delete the listeners from generic class after I moved them to woocommerce/events.php . Fixed in -> https://github.com/Automattic/newspack-network/pull/123/commits/2adaaea5d5603a186195885d0bea86db2fd536ba . This was certainly what was causing the [duplicate events](https://github.com/Automattic/newspack-network/pull/122#pullrequestreview-2253961847)